trogdor/src/main/java/org/apache/kafka/trogdor/workload/TimestampRecordProcessor.java

```diff
@@ -38,7 +38,7 @@
  *
  * Example spec:
  * {
- *    "type": "timestampRandom",
+ *    "type": "timestamp",
```

Comment:
Is this example correct?  In PayloadGenerator.java I see timestampRandom, timestampConstant, gaussianTimestampRandom, gaussianTimestampConstant, but I do not see any type that is simply "timestamp"

Reply:
This is not for a `PayloadGenerator` object, this is for a `RecordProcessor` object.